src/script/lua_api/l_server.cpp

```diff
@@ -45,6 +45,17 @@ int ModApiServer::l_get_server_status(lua_State *L)
 	return 1;
 }
 
+// print(text)
+int ModApiServer::l_print(lua_State *L)
```

Comment:
I'm a bit concerned about the potential consequences of having a Lua API that only has any effect with a certain compile option...